web/content/js/versions.js

```diff
@@ -167,6 +167,86 @@ const navVersions = [
       }
     ]
   },
+  {
+    title: "Foreman 3.4 - Katello 4.6 (supported)",
```

Comment:
This should not be supported yet
```suggestion
    title: "Foreman 3.4 - Katello 4.6 (release candidate)",
```